### DISCUSSION TITLE
Mail is not working

### DISCUSSION THREAD

**Discussion by davide1295** [firefly-iii/firefly-iii#5966] [Firefly III] [answered]
Hi, I'm trying to set up the mail account but I can't get it to work. I set everything up correctly but I get the following error:

`Errore! Possible email error: Failed to authenticate on SMTP server with username "personale@metamorfeo.com" using the following authenticators: "LOGIN", "PLAIN". Authenticator "LOGIN" returned "Expected response code "235" but got code "535", with message "535 Incorrect authentication data".". Authenticator "PLAIN" returned "Expected response code "235" but got code "535", with message "535 Incorrect authentication data".".`

My mail requires the following configuration:

_Secure SSL / TLS Settings (Recommended)
Username: xxx@xxx.com
Password: Use the password of the email account.
Incoming server: mail.xxx.com
IMAP Port: 993 POP3 Port: 995
Outgoing mail server: mail.xxx.com
SMTP Port: 465
IMAP, POP3, and SMTP require authentication._

The configuration in the .env file is as follows:

_MAIL_DRIVER=smtp
MAIL_HOST=mail.xxx.com
MAIL_PORT=465
MAIL_FROM=xxx@xxx.com
MAIL_USERNAME=xxx@xxx.com
MAIL_PASSWORD=********************
MAIL_ENCRYPTION=tls_

I can't understand what the problem is

**Comment by JC5** [comment, marked as answer]
> Incorrect authentication data

Password or username are no good. Sometimes `=` or `"` or `'` in the password will not work. Could it be you have those in your password?

**Reply by davide1295** [reply]
Actually my password contains special characters, I changed my password and now it works!
Thanks.
